src/org/exist/security/SecurityManager.java

```diff
@@ -108,10 +112,6 @@
    @Deprecated
    Subject getSubjectBySessionId(String sessionid);
 
-   void addGroup(int id, Group group);
```

Comment:
why is it require methods renaming?

Reply:
Good question! I don't know - it was a change that you made in your original PR - https://github.com/eXist-db/exist/pull/1056/files#diff-1ae708c6e5ca91cc9ddcfa7181910ca4L111

Reply:
it must stay `addGroup`

Reply:
Why?

Reply:
because it's api change

Reply:
That's fine. This PR already changed the format of the backup, so I consider that it already has API changes and therefore will go in 5.0.0

Reply:
anyway, is it possible to rename it back ... I don't want to deal with another api "renaming"